### PR TITLE
Clean up hacks used to be making Tahoe lighter

### PR DESCRIPTION
**Pull request by OmarIthawi**
Now all the services run again.
To enable elastic search on devstack to test issues related to `ENABLE_COURSE_DISCOVERY`.

I've deprecated `tahoe.up` in favor of `dev.up` since we're not going to merge enterprise anytime soon with this branch.

### TODO

 - [x] Get a review
 - [ ] Get another review
 - [ ] Check that the team (Matej, Maxi and John) knows we're going to run the additional services

### CODE REVIEW

File: docker-compose-tahoe.yml [hunk 1]
```diff
@@ -7,9 +7,6 @@ services:
   chrome:
     image: hello-world
     command: /hello
-  elasticsearch:
```
Comment (johnbaldwin) [inline docker-compose-tahoe.yml:10]: how does removing this enable search on devstack?

**Review by thraxil — APPROVED** [review]
We should probably have some better documentation somewhere explaining why we have a compose file full of hello world images.

File: docker-compose-tahoe.yml (hunk 1 above)
Comment (OmarIthawi) [reply docker-compose-tahoe.yml:10]: Good question: `elasticsearch: image: hello-world` was effectively turning off `elasticsearch` by making it doing nothing.

This is an old hack that is confusing, I've now cleaned it up.

**Comment by OmarIthawi** [comment]
> We should probably have some better documentation somewhere explaining why we have a compose file full of hello world images.

Thanks @thraxil, you're right. The documentation existed as an inline comment. That was difficult to find. I think we don't need this hack anymore so I've restructured this PR to clean up the hack.

Please take a look. I also need to check with Matej, Maxi and John to make sure they're aware of the change.

**Comment by OmarIthawi** [comment]
@melvinsoft @johnbaldwin @grozdanowski Are you all using the cloud devstack (aka Sultan) these days? If you're not, then what do you think of reopening the services above?

The reason we chose to stop them was to avoid having nasty devstack errors on Mac. The attempts weren't successful so we opted out for Sultan so Mac users can use devstack.

**Review by thraxil — APPROVED** [review]

**Comment by grozdanowski** [comment]
@OmarIthawi I use exclusively the local devstack. To be completely honest, have never tried Sultan nor I know even how to :D

**Comment by OmarIthawi** [comment]
Thanks @grozdanowski. Then heads up! If this ruins your experience, we can revert it quickly.

Do you still run of the ancient `omar/quickstart` branch?

**Pull request merged**